wordpress-mobile/WordPress-FluxC-Android
# Woo: Shipping rates endpoint

This PR adds the support for retrieving shipping rates.

**To test:**
1. Go to Woo -> Shipping labels -> Select site
2. Tap on Get shipping rates
3. Wait for the data to be loaded
4. Enter package size parameters
5. Notice a response is printed in the log

**Unit tests:**

Run **WCShippingLabelStoreTest** tests.
